### ISSUE TITLE
New camera is wobbling strangely to front/back on panning

### ISSUE THREAD

**Issue by kungfooman** [playcanvas/model-viewer#169]
![cameramovement](https://user-images.githubusercontent.com/5236548/162608564-79f3000a-d0cf-420b-a5f0-5527fe7283a1.gif)

I am just panning with hold-right-mouse which should move the camera up/down and left/right, but its goings strangely front/back too.

It feels nearly impossible to properly use it or look at some details of some models, because it behaves so strangely.

**Comment by slimbuck** [comment]
Hi @kungfooman, sorry about this! Does this happen with all models or only specific ones? Which machine are you on?

**Comment by slimbuck** [comment]
And just to clarify, only the camera smoothing was changed recently. The basic movement hasn't been changed, so this is def a bug.

**Comment by slimbuck** [comment]
Actually, could you try reloading the model-viewer with browser cache disabled? It shouldn't be cached, but probably worth checking.

**Comment by kungfooman** [comment]
Hi @slimbuck thank you for your quick response!

This issue appeared only in Chrome (Firefox and Edge/Chromium-engine both worked normally).

I reloaded Chrome with DevTools open (which reloads everything based on Chrome setting) multiple times, with no change.

That made me wonder why it works normally in Edge but not in Chrome (same engine), so I compared the settings. And indeed it was the fov settings, that was saved in `localStorage`.

A high fov makes the camera seemingly going back/front by just panning up/down:

![camerafov](https://user-images.githubusercontent.com/5236548/162611818-be31e9f8-c5aa-419b-9404-7e8cb3f13aab.gif)
